### ISSUE TITLE
Cannot read properties of undefined (reading 'STORYBOOK')

### ISSUE THREAD

**Issue by RBlanchet** [nuxt-modules/storybook#637]
Hello 🙌,

I am encountering an issue with the Storybook module in a newly created fresh Nuxt project.

## Steps to Reproduce:
1. Create a new Nuxt project (actually 3.11.2) using the command: `npx nuxi@latest init test-nuxt`
2. Add the Storybook module version 8.0.0 using: `npx nuxi@latest module add storybook`
3. Run the development server with: `yarn run dev`

## Error
When running the development server, the following error is displayed

```shell
Cannot start nuxt:  Cannot read properties of undefined (reading '__STORYBOOK__') 
```

## Workaround:
Manually changing the property `import.meta.env` to `process.env` in the `module.mjs` file resolves the issue and the application runs successfully.

It seems to be potentially linked to this development https://github.com/nuxt-modules/storybook/pull/512 in module 8.0.0

Could you please provide guidance on how to resolve this issue without manually modifying the module file?

Thank you for your support !

**Comment by sifferhans** [comment]
This is a duplicate of #538 I think 😁

**Comment by seancrater** [comment]
I'm having this issue as well

**Comment by mehcode** [comment]
I think we're all waiting on a patch release of the module. 🙏 @chakAs3

**Comment by mehcode** [comment]
It works if you use the git sha in your package.json but pre-pack is then failing in github actions for me, so going to look at another option.

**Comment by seancrater** [comment]
I can't seem to get the suggested workaround to work with Yarn either

**Comment by mehcode** [comment]
@seancrater et all, I pushed a git dep that's pre-built here - `github:mehcode/storybook#bundle`, I find that some times the all the bundling makes git deps fail 

This dep in my `devDependencies` makes it work for me:

```
"@nuxtjs/storybook": "github:mehcode/storybook#bundle",
```
